src/app/components/videochat/jh-video-chat.directive.js

```diff
@@ -222,18 +222,21 @@
         var w = div.innerWidth();
         var h = div.innerHeight();
         var feedW = thumbnailWidth($(".thumb", div).size(), h, w);
+        if (feedW == 64) {
```

Comment:
I think this should be documented in code that it is just workaround for broken firefox implementation of auto, which basically do auto hidding